File: API/Models/Folha.cs
using System;

namespace API.Models
{
    public class Folha
    {
        public Folha() => CriadoEm = DateTime.Now;

        public int Id { get; set; }
        public int HoraTrabalhada { get; set; }
        public int ValorHora { get; set; }
        public int Mes { get; set; }
        public int Ano { get; set; }
        public Funcionario Funcionario { get; set; }
        public int FuncionarioId { get; set; }
        public DateTime CriadoEm { get; set; }
    }
}

File: FRONT/src/app/models/folha.ts
import { Funcionario } from "./funcionario";

export interface Folha {
    id?: number;
    horaTrabalhada: number;
    valorHora: number;
    mes: number;
    ano: number;
    funcionario?: Funcionario;
    funcionarioId: number;
    criadoem?: string;
}
